test/integration/node/lifecycle_test.go

```diff
@@ -297,3 +303,30 @@ func TestTaintBasedEvictions(t *testing.T) {
 		})
 	}
 }
+
+// WaitForPodCondition waits a pods to be matched to the given condition.
+func WaitForPodCondition(c clientset.Interface, ns, podName, desc string, timeout time.Duration, condition podCondition, t *testing.T) error {
```

Comment:
if it is only used here,private is better

Reply:
emm.. it seems my refactor is not be saved.